src/infra/flow-system/sqlite/src/sqlite_flow_trigger_event_store.rs

```diff
@@ -0,0 +1,270 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use chrono::Utc;
+use database_common::{TransactionRef, TransactionRefT};
+use dill::*;
+use futures::TryStreamExt;
+use kamu_flow_system::*;
+use opendatafabric::DatasetID;
+use sqlx::{FromRow, QueryBuilder, Sqlite};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, sqlx::FromRow, PartialEq, Eq)]
+struct EventModel {
+    event_id: i64,
+    event_payload: sqlx::types::JsonValue,
+}
+
+#[derive(Debug, FromRow)]
+struct ReturningEventModel {
+    event_id: i64,
+}
+
```

Comment:
Consider placing this definition into some shared location, maybe `database_common`. It looks common enough to be applied to any event sourcing based aggregate regardless of the database.